extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -575,7 +575,7 @@ function arrayunion( &$parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
 *    see: http://www.php.net/manual/en/function.array-diff.php
 */
 
-    function arraydiff( &$parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
+    function arraydiff( $parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
         if (!isset($arrayid_new) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)